src/main/java/net/openhft/chronicle/core/io/AbstractCloseable.java

```diff
@@ -43,7 +43,9 @@ public abstract class AbstractCloseable implements ReferenceOwner, ManagedClosea
     @Deprecated(/* remove in x.25 */)
     protected static final boolean DISABLE_THREAD_SAFETY = DISABLE_SINGLE_THREADED_CHECK;
     protected static final boolean DISABLE_DISCARD_WARNING = Jvm.getBoolean("disable.discard.warning", false);
-    protected static final boolean STRICT_DISCARD_WARNING = Jvm.getBoolean("strict.discard.warning", false);
+    @SuppressWarnings("DeprecatedIsStillUsed")
+    @Deprecated(/* remove in x.25 */)
+    protected static final boolean STRICT_DISCARD_WARNING;
```

Comment:
I don't understand the purpose of this, when true it seemed to use `Jvm.warn` instead of `Slf4jExceptionHandler.WARN` to log the warning that something was discarded without being closed. Using the latter made it hard to test that we warned when that happened, so I just removed this. Happy to un-remove if there's a good reason for its existence.

Reply:
The purpose was that Jvm.warn() are checked in tests and but Slf4j..WARN isn't; 
We have another means of ignoring warnings, which could be ignored in the XxxxTestCommon instead.